classroom/connect_service.py

```diff
@@ -37,4 +37,4 @@ def create_service():
 
     service = build('classroom', 'v1', credentials=creds,
                     cache_discovery=False)
-    return(service)
+    return (service)

```

Comment:
What is the purpose of the parenthesis?

This should be:

    return service

`return` is not a function.